hbase-server/src/test/java/org/apache/hadoop/hbase/replication/TestMasterReplication.java

```diff
@@ -200,8 +198,8 @@ public void testHFileCyclicReplication() throws Exception {
       // Load 100 rows for each hfile range in cluster '0' and validate whether its been replicated
       // to cluster '1'.
       byte[][][] hfileRanges =
-          new byte[][][] { new byte[][] { Bytes.toBytes("aaaa"), Bytes.toBytes("cccc") },
-              new byte[][] { Bytes.toBytes("ddd"), Bytes.toBytes("fff") }, };
+        new byte[][][] { new byte[][] { Bytes.toBytes("aaaa"), Bytes.toBytes("cccc") },
```

Comment:
All these indent changes intentional? (painful for backporting)

Reply:
Yes, these are check style fixes, since I changed the file, I though of fixing these.